### DISCUSSION TITLE
runsettings not ignoring test project

### DISCUSSION THREAD

**Discussion by jbockle** [coverlet-coverage/coverlet#1295] [Q&A] [answered]
Hi, repro here https://github.com/jbockle/coverlet.csharp10.issue

I can't seem to get VS2022 to honor the runsettings file, I have the `<RunSettingsFilePath>` property specified, am I doing something wrong?

If I run `dotnet test` through command line, it picks up the `.runsettings` properly and reports coverage correctly

**Comment by MarcoRossignoli** [comment, marked as answer]
Can you open the ticket to https://github.com/microsoft/vstest/issues it's unrelated to coverlet, anyway if you enable the logs it's possible understand if your runsetting is used correctly https://github.com/Microsoft/vstest-docs/blob/main/docs/troubleshooting.md#visual-studio

Remember to not attach logs to the issue if it contains private information, in that case you should open a ticket in here https://developercommunity.visualstudio.com/home and attach as private.
